### PR TITLE
Edit: Getting-coins

### PR DESCRIPTION
**Pull request by Mocab**

### CODE REVIEW

File: _posts/falix/general/2021-09-20-getting-coins.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title:  "Getting Coins"
+categories: Falix
+tags: fxgeneral
+permalink: /falix/general/getting-coins/
+---
+
+There are currently two ways of getting coins, you can find them [here](https://client.falixnodes.net/coins)
```
Comment (DaddyChimken) [inline _posts/falix/general/2021-09-20-getting-coins.md:9]:
```suggestion
There are currently two ways of getting coins, you can find them [here](https://client.falixnodes.net/coins).
```

File: _posts/falix/general/2021-09-20-getting-coins.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+# Watching Advertisements
+You can watch advertisements on our Android app to earn coins, follow the steps below to learn how:
+
+1. Make sure that your Falix account is linked with your discord account, you can do so [here](https://client.falixnodes.net/profile/settings).
+2. Download the app from the [Play Store](https://play.google.com/store/apps/details?id=net.falixnodes.falixcoins).
+3. Copy your discord ID, follow [this](https://www.remote.tools/remote-work/how-to-find-discord-id) guide to learn how.
```
Comment (DaddyChimken) [inline _posts/falix/general/2021-09-20-getting-coins.md:16]: Either change this hyperlink to the "Account Management" article, or the Discord support article https://support.discord.com/hc/en-us/articles/206346498.

File: _posts/falix/general/2021-09-20-getting-coins.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+4. Open the app and insert your discord ID into the field given.
+5. Click on "Watch ads".
+
+> Note: any kinds of scripts, macros, and auto clickers are strictly not allowed.
+
+If you do not see any ads, try again later
```
Comment (DaddyChimken) [inline _posts/falix/general/2021-09-20-getting-coins.md:22]:
```suggestion
If you do not see any ads, try again later.
```

File: _posts/falix/general/2021-09-20-getting-coins.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+---
+
+There are currently two ways of getting coins, you can find them [here](https://client.falixnodes.net/coins).
+
+# Watching Advertisements
+You can watch advertisements on our Android app to earn coins, follow the steps below to learn how:
+
+1. Make sure that your Falix account is linked with your discord account, you can do so [here](https://client.falixnodes.net/profile/settings).
+2. Download the app from the [Play Store](https://play.google.com/store/apps/details?id=net.falixnodes.falixcoins).
+3. Copy your discord ID, follow [this](https://support.discord.com/hc/en-us/articles/206346498) guide to learn how.
```
Comment (DaddyChimken) [inline _posts/falix/general/2021-09-20-getting-coins.md:16]:
```suggestion
3. Copy your Discord user ID, follow [this](https://support.discord.com/hc/en-us/articles/206346498) guide to learn how.
```

File: _posts/falix/general/2021-09-20-getting-coins.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+4. Open the app and insert your discord ID into the field given.
```
Comment (DaddyChimken) [inline _posts/falix/general/2021-09-20-getting-coins.md:17]:
```suggestion
4. Open the app and insert your discord user ID into the given field.
```

**Review by DaddyChimken — APPROVED** [review]

**Review by itsjuoum — CHANGES_REQUESTED** [review]

**Review by itsjuoum — APPROVED** [review]

**Pull request merged**